docs/reference/expectations.rst

```diff
@@ -43,6 +43,12 @@ usage (e.g. during refactoring). See examples in a later section.
 
 .. code-block:: php
 
+    shouldNotReceive(method_name)
+
+Declares that the mock should not expect a call to the given method name.
```

Comment:
I think a little comment here to say that it is effectively shorthand for `shouldReceive()->never()` would be good.